CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
 - `MouseButton` is now non-exhaustive.
 - Remove `Other` and add `Forward` and `Back` variants to `MouseButton`. Use the new `PointerButtonEvent::button_code` in place of `Other`.
 - `GrabStartData` has been renamed to `PointerGrabStartData`
+- Added `TouchHandle` for Wayland client touch support (see `Seat::get_touch`)
```

Comment:
Technically, this should be under [Additions > Clients & Protocols](https://github.com/Smithay/smithay/blob/032c0de390d17219f53065bb69c99d5ef1762974/CHANGELOG.md#clients--protocols-1) as this is not "Braking Change" per se